src/util/name.h

```diff
@@ -223,6 +223,7 @@ class name {
     struct ptr_eq { bool operator()(name const & n1, name const & n2) const { return n1.m_ptr == n2.m_ptr; } };
 };
 
+unsigned hash(name const & n);
```

Comment:
```suggestion
inline unsigned hash(name const & n) { return n.hash(); }
```

This is the kind of thing you want to inline.  (`inline` keyword has of course nothing to do with inlining.)